platform/onebox/backup_and_restore/registry/export.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
```

Comment:
rename the file to a better name like export_images.sh

Reply:
why, we have kept it in separate folders.
So we will have
regsitry/(ex/im)port.sh
storage/(ex/im)port.sh
packages/(ex/im)port.sh
datasets/(ex/im)port.sh